README.md

```diff
@@ -1,31 +1,69 @@
-# Reconstruction pipeline
-
-This pipeline is offering a solution to build full brain network models starting from standard structural MR scans.
-It is used to preprocess the MR scans in order to get actual files that are compatible with TVB. The result can be later uploaded in TVB or used independently for modeling. 
+<h1 align="center">
+  <br>
+  <a href="http://www.amitmerchant.com/electron-markdownify"><img src="https://cdn.evbuc.com/eventlogos/50084740/tvbtaglinecrop.jpg" alt="The Virtual Brain" width="300"></a>
+  <br>
+  Reconstruction pipeline
+  <br>
+</h1>
+<h4 align="center">This pipeline is offering a solution to build full brain network models starting from standard structural MR scans.</h4>
+<p align="center">
+  <a href="https://pegasus.isi.edu/downloads/">
+    <img src="https://img.shields.io/badge/Pegasus%20WMS-4.9.3-%2349A13D"
+         alt="Gitter">
+  </a>
+  <a href="https://neurostars.org/">
+	  <img src="https://img.shields.io/badge/Chat-on%20Neurostars-%23673B90">
+</a>
+  <a href="https://saythanks.io/">
+      <img src="https://img.shields.io/badge/SayThanks.io-%E2%98%BC-1EAEDB.svg">
+  </a>
+  <a href="https://www.paypal.com/">
+    <img src="https://img.shields.io/badge/%24-Donate-%239D2E2B">
+  </a>
+</p>
+
+<p align="center">
+  <a href="#Folder-structure">Folder structure</a> •
+  <a href="#Data-structure">Data structure</a> •
+  <a href="#How-to-launch">How to launch</a> •
+  <a href="#How-to-rerun">How to rerun</a> •
+  <a href="#Environment">Environment</a> •
+  <a href="#Contributors">Contributors</a>
+</p>
+<p>
+<hr>
+
+It is used to preprocess the MR scans in order to get actual files that are compatible with TVB. The result can be later uploaded in TVB or used independently for modeling.
 
 The mandatory inputs are DWI and T1 scans. Optionally, CT scans can be given as input, if sensors preprocessing is needed.
 
-We are using the [Pegasus WMS](https://pegasus.isi.edu/) in order to connect and automatize the pipeline steps. Pegasus is distributed under the Apache v2.0 license.
+We are using the <a href="https://pegasus.isi.edu/">Pegasus WMS</a> in order to connect and automatize the pipeline steps. Pegasus is distributed under the Apache v2.0 license.
+</p>
+
+<p align="center">
+  <img height="600" src="https://ars.els-cdn.com/content/image/1-s2.0-S1053811915002505-gr1.jpg">
+</p>
```

Comment:
This image should be clearly cited. I believe it is from article [2] in the Biobliography.

Reply:
Yes, it is from the article [2] in the Bibliography. I thought it is representing the process clearly that's why I placed it here. Should I replace it? Also, ma'am, if you have any other image then please share it here.

Reply:
I also think it is a good image. I only suggested to have written close to the image where it is taken from

Reply:
Okay ma'am

Reply:
We tend to be less formal in our conversation. 
Thus, no need to address me with ma'am ;-)

Reply:
Okay sure :D